api-svc/raid-api/src/test/java/au/org/raid/api/service/datacite/DataciteServiceTest.java

```diff
@@ -1,164 +1,60 @@
 package au.org.raid.api.service.datacite;
 
-import au.org.raid.api.factory.DatacitePayloadFactory;
+import au.org.raid.api.factory.datacite.DataciteDtoFactory;
+import au.org.raid.api.model.datacite.DataciteData;
+import au.org.raid.api.model.datacite.DataciteDto;
 import au.org.raid.api.spring.config.DataciteProperties;
-import au.org.raid.idl.raidv2.model.*;
-import com.fasterxml.jackson.databind.JsonNode;
+import au.org.raid.idl.raidv2.model.RaidCreateRequest;
+import au.org.raid.idl.raidv2.model.Title;
+import au.org.raid.idl.raidv2.model.TitleType;
+import au.org.raid.idl.raidv2.model.TitleTypeWithSchemaUri;
+import com.fasterxml.jackson.core.JsonProcessingException;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import static org.junit.jupiter.api.Assertions.*;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import static org.mockito.Mockito.*;
-import java.net.HttpURLConnection;
-import java.net.URL;
-import java.io.OutputStream;
-import java.io.InputStream;
-
-import java.lang.reflect.Array;
-import java.lang.reflect.Method;
-import java.util.HashSet;
-import java.util.Set;
-import org.junit.jupiter.api.Test;
-
-import jakarta.json.Json;
 import org.mockito.Mock;
+import org.springframework.web.client.RestTemplate;
 
-import java.io.IOException;
-import java.io.StringReader;
-import java.net.URISyntaxException;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-import java.util.Objects;
+import java.util.List;
 
-import java.lang.reflect.Method;
-import java.util.HashSet;
-import java.util.Set;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 
 public class DataciteServiceTest {
```

Comment:
This doesn't really test the DataciteService anymore. Would be good to rename this and add a new test for the DataciteService.